contracts/Receivable.sol

```diff
@@ -24,28 +24,34 @@ contract Receivable is
     ERC721EnumerableUpgradeable,
     ERC721BurnableUpgradeable,
     UUPSUpgradeable,
-    Governable,
     IReceivable
 {
+    address public governor;
+
+    /*//////////////////////////////////////////////////////////////
+                               MODIFIERS
+    //////////////////////////////////////////////////////////////*/
+
+    modifier onlyGovernor() virtual {
+        if (msg.sender != governor) {
+            revert Errors.CallerNotGovernor({ governor_: governor, caller_: msg.sender });
+        }
+        _;
+    }
     /*//////////////////////////////////////////////////////////////
                              UUPS FUNCTIONS
     //////////////////////////////////////////////////////////////*/
 
     function _authorizeUpgrade(address newImplementation) internal override onlyGovernor { }
 
-    /// @custom:oz-upgrades-unsafe-allow constructor
-    // constructor() {
-    // _disableInitializers();
-    // }
-
     /// @inheritdoc IReceivable
-    function initialize(address initialGovernor_) external override initializer {
+    function initialize(address isleGlobal_) external override initializer {
```

Comment:
Here could you add zero address check to prevent potential address(0) issue? You can add a general address(0) error, so that I can use it in my PR.